README.md

```diff
@@ -6,18 +6,26 @@
       * [Usage](#usage)
          * [Basic configuration parameters](#basic-configuration-parameters)
          * [Example configurations](#example-configurations)
-            * [Kernel driver config](#kernel-driver-config)
-            * [Advanced kernel driver config](#advanced-kernel-driver-config)
-            * [DPDK userspace driver config](#dpdk-userspace-driver-config)
+            * [Basic config](#basic-config)
+            * [Extended config](#extended-config)
          * [Advanced configuration](#advanced-configuration)
       * [Contributing](#contributing)
 
 # Accelerated Bridge CNI plugin
-This plugin enables the configuration and usage of Accelerated Bridge VF networks in containers and orchestrators like Kubernetes.
+This plugin allows using Linux Bridge with hardware offloading in containers and orchestrators like Kubernetes.
+Accelerated Bridge CNI plugin requires NIC with support of [SR-IOV](http://blog.scottlowe.org/2009/12/02/what-is-sr-iov/)
```

Comment:
this link talks about SR-IOV in legacy context which is configuring

Reply:
I agree. This link can confuse. I removed it.